validator/client/validator.go

```diff
@@ -516,27 +516,18 @@ func (v *validator) UpdateProtections(ctx context.Context, slot uint64) error {
 	if err != nil {
```

Comment:
This is fine as this function is only called in the main thread and never concurrently

Reply:
@rauljordan it is being called concurrently go routine at the end keep updating the old data while update protections is being called